### PR TITLE
[skip ci] Enable bazel cache

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Enable the bazel cache to work with kubevirts cache

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ export CDI_NAMESPACE=$NAMESPACE
 echo "namespace: ${NAMESPACE}, cdi-namespace: ${CDI_NAMESPACE}"
 
 readonly ARTIFACTS_PATH="exported-artifacts"
+readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
```
Comment (rmohr) [inline automation/test.sh]: the path of the url should be `kubevirt.io/containerized-data-importer`.

Comment (awels) [reply automation/test.sh]: Fixed

**Comment by rmohr** [comment]
Looks good to me. Follow-ups to get all features from Prow:

* Save your junit ginkgo junit file like this, so that Prow can pick it up: `--junit-output=$ARTIFACTS_PATH/junit.functest.xml`.
* Copy and wire this reporter into your testsuite: https://github.com/kubevirt/kubevirt/tree/master/tests/reporter. This will collect logs per test-failure and uploads them to GCS.

Once you have these two changes in place, we can also enable flakefinder for you.

**Comment by rmohr** [comment]
Oh and ARTIFACTS_PATH should point to $ARTIFACTS (that is the environment variable from prow, which denotes the location for artifacts to upload to GCS).

**Comment by rmohr** [comment]
/lgtm

I am unsure if you want to disable the stdci webhook, or just delete the automation folder. I am mostly thinking about backports here.

**Comment by awels** [comment]
So I think for now I want to remove the automation folder, in case we have a back port we can still run CI on that.

**Comment by awels** [comment]
The failures appear to all be because of snapshot cloning failing in one way or another. Needs more investigation, but proves the artifact generation works properly. So I am happy at this point.

**Pull request merged**